lib/ansible/modules/cloud/amazon/aws_eks_cluster.py

```diff
@@ -36,6 +36,18 @@
       - absent
       - present
     default: present
+  wait:
+    description: >-
+      Specifies whether the module waits until the cluster becomes active after
+      creation. It takes "usually less than 10 minutes" per AWS documentation.
+    type: bool
+    default: 'no'
+  wait_timeout:
+    description: >-
+      The duration in seconds to wait for the cluster to become active. Defaults
+      to 1200 seconds (20 minutes).
+    type: int
```

Comment:
This line is failing CI - just remove it (it makes no sense that `type: int` isn't valid here, but it isn't)

Reply:
Right. Removed.